ios/PalmApp/PalmApp-Mainnet-Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>$(MARKETING_VERSION).$(CURRENT_PROJECT_VERSION)</string>
```

Comment:
revert this please. I don't think this is needed. also, if were to update this, need to update palmapp-testnet-info.plist as well.

Reply:
testnet plist is already set of $(MARKETING_VERSION).$(CURRENT_PROJECT_VERSION)

It was changed because the mainnet was not configured

Reply:
got it.